developer/frameworks/articles/05-developing-fragments/01-creating-a-fragment.markdown

```diff
@@ -0,0 +1,60 @@
+# Creating Fragments [](id=creating-a-fragment)
+
+If you aren't sure about the basics of Fragments and Collections, you should 
+read [Creating Page Fragments](link) first so you know what you're getting 
+yourself into. Once, you're ready, start by creating the Collection:
+
+1.  Go to *Site Administration* and for your selected site click *Build*
+    &rarr; *Fragments*.
+ 
+2.  Create a new Collection named *Developing Fragments*.
+
+Next, create a *Section* and a *Component*.
+
+## Creating a Section
+
+The list of Collections appears on the left in the Page Fragments page.
+
+1.  Ensure that you are in the *Developing Fragments* collection.
+
+2.  Click the [Add](../../../images/icon-add.png) button and select *Section*.
+
+3.  Name your Section "Basic Section"
+
+You're now on the Fragment editing page. There are four panes on this screen.
+You enter HTML in the top left pane, CSS in the top right, JavaScript in the
+bottom left, and preview the results in the bottom right.
+
+You can look at the three editing panes as if each were writing to a separate 
+file. Everything in the HTML pane goes to `index.html`, the CSS pane goes to 
+`index.css`, and the JavaScript one goes to `index.js`. The preview pane renders
+everything as it will look on the page. Here's how this works:
+
+1.  Add the following code inside the HTML pane:
+    **NEED A NEW EXAMPLE**
```

Comment:
We can use one of the contributed section or component as an example, since they were created by us